OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -660,9 +661,7 @@ void TracerThread::Reset() {
 }
 
 void TracerThread::PrintStatsIfTimerElapsed() {
-  constexpr uint64_t EVENT_COUNT_WINDOW_S = 5;
-
-  if (stats_.event_count_begin_ns + EVENT_COUNT_WINDOW_S * 1'000'000'000 <
+  if (stats_.event_count_begin_ns + EVENT_STATS_WINDOW_S * 1'000'000'000 <
```

Comment:
Define 1'000'000'000 as a constant (kNsPerSec?).

Reply:
Done. I used the macro casing though, I know it's against the style guide, but for consistency with the rest of Orbit.